openstack_ansibleee/Dockerfile

```diff
@@ -21,13 +23,13 @@ RUN cd /var/tmp/edpm-ansible && python3 setup.py install --prefix=/usr
 # It will give Device/Resource busy. In order to fix that. Clean up the directory
 # Only when EDPM_LOCAL_REPO is defined.
 RUN if [ "$EDPM_LOCAL_REPO" == "" ] ; then \
-        echo $EDPM_LOCAL_REPO; \
+        echo EDPM_LOCAL_REPO $EDPM_LOCAL_REPO; \
         rm -fr /var/tmp/edpm-ansible; \
     fi
```

Comment:
We can drop all of this... we already use the "local" repo, we just don't need to mount it as en external volume anymore.

Reply:
done